### ISSUE TITLE
Missing Documentation for PR_BODY property

### ISSUE THREAD

**Issue by CalvinRodo** [BetaHuhn/repo-file-sync-action#172]
**🐞 Describe the bug**

The README.md does not have documentation for the PR_BODY property how it works and what the default is. 

Would also be nice to have an example for folks to follow.

**Comment by BetaHuhn** [comment]
Agreed, this was overlooked by me when the option was added with #80.

Will add more documentation when I have time in the next few days.
